docusaurus.config.js

```diff
@@ -6,8 +6,8 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Pemrograman Berbasis Platform',
```

Comment:
Hmm, it seems like Docusaurus does not support i18n or the title and tagline yet. I will try to look into using the Translation context later to fix this, i will create an issue for this later